pkgs/development/libraries/ndi/default.nix

```
@@ -2,6 +2,11 @@
 
 let
   versionJSON = lib.importJSON ./version.json;
+  ndiPlatform =
+    if stdenv.isAarch64 then "aarch64-rpi4-linux-gnueabi"
+    else if stdenv.isAarch32 then "arm-rpi2-linux-gnueabihf"
+    else if stdenv.isi686 then "i686-linux-gnu"
+    else "x86_64-linux-gnu";
```

If we set platforms explicitly, we should probably not fallback to x86_64-linux here.

```suggestion
    else if stdenv.isx86_64 then "x86_64-linux-gnu"
    else throw "....";
```